proposed/openebs/latest/questions.yml

```diff
@@ -1,5 +1,7 @@
 categories:
 - storage
+labels:
+  io.rancher.certified: partner
```

Comment:
Hi @prateekpandey14 can u please share who is your contact person in rancher of the partner network, if not you may also contact us via https://rancher.com/partners/, thanks.

Reply:
/cc @muratkars

Reply:
Hi @guangbochen - our contact is Alex Belak (Director Strategic Alliances).

Reply:
@guangbochen PTAL thanks.